src/Groups/GAPGroupsLibrary.jl

```diff
@@ -0,0 +1,205 @@
+
+export
+    blocks,
+    isperfect,
+    isprimitive,
+    isregular,
+    issemiregular,
+    istransitive,
+    maximal_blocks,
+    number_perfect_groups,
+    number_small_groups,
+    number_transitive_groups,
+    perfect_group,
+    perfect_identification,
+    representatives_minimal_blocks,
+    small_group,
+    small_group_identification,
+    transitive_group,
+    transitive_identification,
+    transitivity
+
+###################################################################
+# Transitive groups
+###################################################################
+
+"""
+    number_transitive_groups(n::Int)
+Return the number of transitive groups acting on a set of size `n`.
+"""
+number_transitive_groups(n::Int) = GAP.Globals.NrTransitiveGroups(n)
+
+"""
+    transitive_group(deg::Int, i::Int)
+Return the `i`-th group in the catalogue of transitive groups over the set {`1`,...,`deg`} in the GAP Small Groups Library. The output is a group of type ``PermGroup``.
+"""
+function transitive_group(deg::Int, n::Int)
+   @assert n<= number_transitive_groups(deg) "There are less than $n transitive groups of degree $deg."
+   return PermGroup(GAP.Globals.TransitiveGroup(deg,n), deg)
+end
+
+"""
+    transitive_identification(G::PermGroup)
+Return (`deg`, `m`), where `G` = transitive_group(`deg`,`m`).
+"""
+transitive_identification(G::PermGroup) = GAP.Globals.TransitiveIdentification(G.X)
+
+"""
+    transitivity(G::PermGroup, L::AbstractVector{Int})
+Return the maximum `k` such that the action of `G` over `L` is `k`-transitive. The output is ``0`` if `G` is not transitive. If `L` is not specified, then `L` is taken as the set of moved points by `G`.
+"""
+transitivity(G::PermGroup, L::AbstractVector{Int}) = GAP.Globals.Transitivity(G.X, GAP.julia_to_gap(L))
+transitivity(G::PermGroup) = GAP.Globals.Transitivity(G.X)
+
+"""
+    istransitive(G::PermGroup, L::AbstractVector{Int})
+Return whether the action of the group `G` on `L` is transitive. If `L` is not specified, then `L` is taken as the set of moved points by `G`.
+"""
+istransitive(G::PermGroup, L::AbstractVector{Int}) = GAP.Globals.IsTransitive(G.X, GAP.julia_to_gap(L))
+istransitive(G::PermGroup) = GAP.Globals.IsTransitive(G.X)
```

Comment:
Hmm, maybe this would be better:
```suggestion
istransitive(G::PermGroup) = GAP.Globals.IsTransitive(G.X, degree(G))
```

Reply:
Same for other related functions

Reply:
after actually testing this, my original suggestion of course was wrong, but I just fixed it